version: "3.11"

services:
  django-app:
    image: harryphoebus/tensor-sydnicate:latest
    platform: linux/amd64
    container_name: ai-drug-app
    restart: always

    environment:
      DEBUG=False  # Set DEBUG to False for production
    command: python manage.py runserver 0.0.0.0:80

    labels:
      - "traefik.enable=true"
      - "traefik.http.services.web-st125064.loadbalancer.server.port=8000"
      - "traefik.http.routers.web-st125064.rule=Host(`st125064.ml.brain.cs.ait.ac.th`)"
      - "traefik.http.routers.web-st125064.tls=true"
      - "traefik.http.routers.web-st125064.tls.certresolver=staging"
      - "traefik.http.routers.web-st125064.entrypoints=websecure"

    networks:
      - default
      - traefik_default

networks:
  default:
  traefik_default:
    external: true
